src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -455,13 +444,8 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
                   <button
                     className="device-toggle-button"
                     css={audioOptionsOpen ? videoControlActiveStyles : videoControlInActiveStyles}
-                    onClick={event => {
-                      const target = event.target as Element;
-                      // We want to ensure to only toggling the menu open or closed when clicking the icon, not the select menu
-                      if (!target.closest('#select-microphone')) {
-                        setAudioOptionsOpen(prev => !prev);
-                      }
-                    }}
+                    onClick={() => setAudioOptionsOpen(prev => !prev)}
+                    onKeyDown={event => handleKeyDown(event, () => setAudioOptionsOpen(prev => !prev))}
```

Comment:
we don't need to filter out the select element in that iteration, both clicks and keypresses to select an option will toggle the menu off